.envrc

```diff
@@ -2,7 +2,11 @@
 
 # if nix is installed, use it
 if [ ! -r .nix-disable  ] && has nix-env; then
-    export NIX_PROFILE=/nix/var/nix/profiles/eng-playbook
+    if [ -w "/nix/var/nix/profiles/per-user/${LOGNAME}" ]; then
+      export NIX_PROFILE="/nix/var/nix/profiles/per-user/${LOGNAME}/eng-playbook"
+    else
+      log_error "Cannot determine writable location for your NIX_PROFILE"
+    fi
```

Comment:
I know I proposed this pattern, but upon further reflection, maybe it would be better to just set the `NIX_PROFILE` and then have nix fail if it's not right. That way we'd probably get better error messages.

Reply:
I can go either way to be honest, but I updated the PR to go with your recommendation.

Reply:
As I proposed, the user will get an error, but it won't tell them which path the script tried to use, which is less than helpful.

Reply:
[I see the `HOWTO.md` got updated](https://github.com/trussworks/Engineering-Playbook/pull/210/commits/6c3f160ad94362ed39f86761a30fab46adde1820), but this must also get updated too since this is where the error would occur :vs: the example for an `.envrc` file.